ui/webpack.common.ts

```diff
@@ -23,6 +23,7 @@ module.exports = {
   resolve: {
     alias: {
       src: path.resolve(__dirname, 'src'),
+      react: path.resolve('./node_modules/react'),
```

Comment:
this'll let us do things like `yarn add ../../clockface/dist/influxdata-clockface-1.0.3.tgz.`

Reply:
could you flesh out an example of this.. ? I was always npm linking clockface and giraffe. I should share how to do that with people now that I think about it.